tests/consumer_pytest.py

```diff
@@ -0,0 +1,14 @@
+from src.consumer import Consumer
+from src.job import Job
+from src.load_profile import LoadProfile
+from src.producer import Producer
+
+
+def test_request_producer():

```

Comment:
Checks that a consumer sending a job gets accepted and added in the producer's list of consumers